third_party/amd/python/triton_amd.cc

```diff
@@ -297,4 +308,10 @@ void init_triton_amd(py::module &&m) {
       arg.addAttr(llvm::Attribute::InReg);
     }
   });
+
+  m.def("link_hsaco",
+        [](const std::string &inPath, const std::string &outPath) {
+          if (!lldInvoke(inPath.c_str(), outPath.c_str()))
+            throw std::runtime_error("couldn't link");
```

Comment:
Can we print out more details regarding the error? Could be helpful when really hiting this.

Reply:
sure